jCOS/createRestrictions/createRestrictions.dockerfile

```diff
@@ -0,0 +1,31 @@
+# Type: Dockerfile
+# Description: Dockerfile for the jCOS createRestrictions component
+
+# Create the build image
+
+FROM maven:3.6.3-jdk-11 AS build
+
+LABEL Khaos Research Group <khaos.uma.es>
+
+WORKDIR /usr/local/src/
+COPY . /usr/local/src/
+
+# Build the component jar file
```

Comment:
Cache the dependencies before building, mvn is very slow

Reply:
Dependencies cached before building. Done